apps/web/src/components/display/UserManagementTable.tsx

```diff
@@ -0,0 +1,100 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import sortIcon from '@assets/img/sort.svg';
+import { buttonBase, buttonColor } from '@components';
+import { EmployeeRO, formatDate } from '@ien/common';
+import { Spinner } from '../Spinner';
+
+export interface UserManagementProps {
+  employees: EmployeeRO[];
+  loading: boolean;
+  onSortChange: (field: string) => void;
+}
+
+export const UserManagementTable = (props: UserManagementProps) => {
+  const { employees, loading, onSortChange } = props;
+  const router = useRouter();
+
+  return (
+    <div className='overflow-x-auto'>
+      <table className='text-left w-full'>
+        <thead className='whitespace-nowrap bg-bcLightGray text-bcDeepBlack'>
+          <tr className='border-b-2 border-yellow-300 text-sm'>
+            <th className='px-6 py-4'>
+              <div className='flex align-middle justify-between'>
+                <span>Name</span>
+                <button id='sort-by-name' onClick={() => onSortChange('name')}>
+                  <img src={sortIcon.src} alt='sort' />
+                </button>
+              </div>
+            </th>
+            <th className='px-6'>Email Address</th>
+            <th className='px-6'>
+              <div className='flex align-middle justify-between'>
+                <span>Created On</span>
+                <button id='sort-by-name' onClick={() => onSortChange('name')}>
+                  <img src={sortIcon.src} alt='sort' />
+                </button>
+              </div>
+            </th>
+            <th className='px-6'>
+              <div className='flex align-middle justify-between'>
+                <span>Role</span>
+                <button id='sort-by-name' onClick={() => onSortChange('role')}>
+                  <img src={sortIcon.src} alt='sort' />
+                </button>
+              </div>
+            </th>
+            <th className=''></th>
+          </tr>
+        </thead>
+        <tbody className='text-bcBlack'>
+          {employees.length === 0 ? (
+            <tr className='text-center shadow-xs text-base font-bold'>
+              <td colSpan={4} className='py-4'>
+                No Results
+              </td>
+            </tr>
+          ) : (
+            employees.map((employee: EmployeeRO) => (
+              <tr
+                key={employee.id}
+                className='text-left shadow-xs whitespace-nowrap even:bg-bcLightGray text-sm '
+              >
+                <td className='pl-6 py-5'>{employee.name}</td>
+                <td className='px-6'>{employee.email}</td>
+                <td className='px-6'>{employee.createdDate && formatDate(employee.createdDate)}</td>
+                <td className='px-6'>{employee.role.toUpperCase()}</td>
+
+                <td className='px-6 text-right'>
+                  {employee.role.toLowerCase() !== 'pending' ? (
```

Comment:
Why can't we change pending role?

Reply:
designs show no change role button next to pending role users